packaging/resource_suite_s3_nocache.py

```diff
@@ -37,24 +47,24 @@ def make_arbitrary_file(f_name, f_size, buffer_size=32*1024*1024):
     # do not care about true randomness
     random.seed(5)
     bytes_written = 0
-    buffer = buffer_size * ['x']
+    buffer = buffer_size * [0x78] # 'x'
```

Comment:
adding a comment about why the hex form is used instead of `'x'` could be very helpful for new readers of the source code.